committee-steering/governance/README.md

```diff
@@ -19,6 +19,25 @@ Specific attention has been given to:
 - Process for agreeing upon technical decisions
 - Process for ensuring technical assets remain healthy
 
+## SIG Charter approval process
+
+When introducing a SIG charter or modification of a charter the following process should be used.  
+As part of this we will define roles for the OARP process (Owners, Approvers, Reviewers, Participants)
```

Comment:
Worth linking to an explanation of OARP?

Or, they could just search, I suppose. The top hit is reasonable?
https://stumblingabout.com/tag/oarp/

Reply:
This this is governance related I think it's worth being explicit and not leaving it up to the reader to work it out. A direct link would be useful. Just my 2 cents.